src/main/kotlin/no/nav/familie/ks/sak/kjerne/beregning/PeriodeResultat.kt

```diff
@@ -0,0 +1,89 @@
+package no.nav.familie.ks.sak.kjerne.beregning
+
+import no.nav.familie.ks.sak.common.exception.FunksjonellFeil
+import no.nav.familie.ks.sak.common.tidslinje.Periode
+import no.nav.familie.ks.sak.common.tidslinje.tilTidslinje
+import no.nav.familie.ks.sak.common.tidslinje.utvidelser.kombinerTidslinjer
+import no.nav.familie.ks.sak.common.tidslinje.utvidelser.tilPerioder
+import no.nav.familie.ks.sak.common.util.TIDENES_ENDE
+import no.nav.familie.ks.sak.common.util.TIDENES_MORGEN
+import no.nav.familie.ks.sak.common.util.sisteDagIMåned
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.PersonResultat
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.Resultat
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.UtdypendeVilkårsvurdering
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.Vilkår
+import no.nav.familie.ks.sak.kjerne.personident.Aktør
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonType
+import java.math.BigDecimal
+import java.time.LocalDate
+
+data class PeriodeResultat(
+    val aktør: Aktør,
+    val periodeFom: LocalDate?,
+    val periodeTom: LocalDate?,
+    val vilkårResultater: Set<PeriodeVilkår>
+) {
+    fun allePåkrevdeVilkårErOppfylt(personType: PersonType): Boolean =
+        vilkårResultater.map { it.vilkårType }.containsAll(Vilkår.hentVilkårFor(personType)) &&
+            vilkårResultater.all { it.resultat == Resultat.OPPFYLT }
```

Comment:
Her må vi godta både `OPPFYLT` og `IKKE_AKTUELT`. Dersom resultat er `IKKE_AKTUELT` betyr det kun at vilkåret ikke har noe å si for retten til kontantstøtte. Det betyr ikke at vilkåret er `IKKE_OPPFYLT` 👍 Dette retter bugen https://favro.com/organization/98c34fb974ce445eac854de0/077068028bffba85055cca2d?card=Tea-10265

Reply:
fikset det